base/redhat-8/install.sh

```diff
@@ -26,14 +26,14 @@ microdnf -y --nodocs install glibc-langpack-en
 #We get around the gen above by forcing the language install, and then point to it.
 export LANG=en_US.utf8
 
-microdnf -y --nodocs install wget sudo shadow-utils procps
+microdnf -y --nodocs install wget sudo shadow-utils procps tar
 #install busybox direct from the multiarch since epel isn't availible yet for redhat8
 wget -O /bin/busybox https://busybox.net/downloads/binaries/1.28.1-defconfig-multiarch/busybox-`arch`
 chmod +x /bin/busybox
-microdnf -y --nodocs install gcc redhat-rpm-config python2-devel libffi-devel openssl-devel tar
+microdnf -y --nodocs install python2-pip python2-devel redhat-rpm-config gcc libffi-devel openssl-devel
 pip2 --no-cache-dir install requests ansible
```

Comment:
Is it necessary to use pip for these? Are there no rpm packages available? Perhaps we should try pushing this upstream to Red Hat?

Reply:
I found the requests package at `microdnf install python2-requests` but none for ansible as of yet. Switching to `python2-requests` does not make much of a difference compared to the pip installed one:
```
base-redhat-8               latest         38feade3e92c       About a minute ago   334MB
base-redhat-8-piprequests   latest         ac651e75a3a5       3 minutes ago        334MB
```

I guess the only benefit would be to remove pip entirely for any image size savings, but we may still need it for ansible. I guess if we really wanted, we would also remove pip after we've installed the relevant packages?

Reply:
Agree not worth it for just requests.. I think best approach (longer term) would be to push this request to Red Hat, and see if they could include rpm for ansible or alternatively include the s390 binaries for cryptography, so that the pip command doesn't require gcc etc. and building it from source.